### ISSUE TITLE
Critical DNS Updates

### ISSUE THREAD

**Issue by mikeln** [samsung-cnct/kraken-lib#866]
PER: https://github.com/samsung-cnct/issues/issues/63

After examining the way kubernetes/cluster/addons/dns was changed for version 1.5, 1.6, 1.7:

update: https://github.com/samsung-cnct/kraken-lib/blob/master/ansible/roles/kraken.services/defaults/main.yaml

kubedns_image_v16: "gcr.io/google_containers/k8s-dns-kube-dns-amd64:1.14.5"

dnsmasq_image_v16: "gcr.io/google_containers/k8s-dns-dnsmasq-nanny-amd64:1.14.5"
dnsmasq_image_v15: "gcr.io/google_containers/k8s-dns-dnsmasq-amd64:1.14.5"

dnsmasq_metrics_image_v16: "gcr.io/google_containers/k8s-dns-sidecar-amd64:1.14.5"

v17, v18 point at v16 (default) anymore.

**Comment by mikeln** [comment]
for currently running clusters... you can temporarily override the version by setting the following in your config.yaml

```
definitions:
  dnsConfig:
    - &defaultDns
      name: defaultDns
      kind: dns
      kubedns:
        cluster_ip: 10.32.0.2
        dns_domain: cluster.local
        namespace: kube-system
       kubedns_image_v16: "gcr.io/google_containers/k8s-dns-kube-dns-amd64:1.14.5"
      dnsmasq_image_v16: "gcr.io/google_containers/k8s-dns-dnsmasq-nanny-amd64:1.14.5"
      dnsmasq_image_v15: "gcr.io/google_containers/k8s-dns-dnsmasq-amd64:1.14.5"
      dnsmasq_metrics_image_v16: "gcr.io/google_containers/k8s-dns-sidecar-amd64:1.14.5"
```

**Comment by davidewatson** [comment]
A fix for new clusters has been merged: https://github.com/samsung-cnct/kraken-lib/pull/867

**Comment by coffeepac** [comment]
fix of the fix:  #870
